docs/content/preview/explore/linear-scalability/sharding-data.md

```diff
@@ -130,57 +138,56 @@ CREATE TABLE order_details (
 );
 ```
 
-**In YCQL**, you can't create tables with range sharding. YCQL tables are always hash sharded.
+In YCQL, you can't create tables with range sharding. YCQL tables are always hash sharded.
 
-## Try it out
+## Examples
 
 This example shows how automatic sharding works in YugabyteDB. First, you create some tables to understand how automatic sharding works. Then, you insert entries one by one, and examine how the data gets distributed across the various nodes.
 
 This tutorial uses the [yugabyted](../../../reference/configuration/yugabyted/) cluster management utility.
 
-### Create a cluster
+### Create a universe
 
-To create a cluster, do the following:
+To create a universe, do the following:
 
-1. Create a single node cluster by running the following command:
+1. Create a single-node universe by running the following command:
 
     ```sh
-    $ ./bin/yugabyted start \
+    ./bin/yugabyted start \
                      --base_dir=/tmp/ybd1 \
                      --listen=127.0.0.1 \
                      --tserver_flags "memstore_size_mb=1"
     ```
 
     `memstore_size_mb=1` sets the total size of memstores on the tablet-servers to `1MB`. This will force a flush of the data to disk when a value greater than 1MB is added, so that you can observe which tablets the data is written to.
 
```

Comment:
probably need reminder about adding loopback addresses on macos - need them for addresses 127.0.0.2-6

Reply:
I'm not sure what you mean. I didn't change any technical content in this file  (wasn't in scope for this ticket)